tests/reporter/kubernetes.go

```diff
@@ -619,6 +622,17 @@ func (r *KubernetesReporter) logSRIOVOperatorConfigs(virtCli kubecli.KubevirtCli
 	r.dumpK8sEntityToFile(virtCli, sriovOperatorConfigsEntity, v1.NamespaceAll, operatorConfigPath)
 }
 
+func (r *KubernetesReporter) logNetworkAttachmentDefinitionInfo(virtCli kubecli.KubevirtClient) {
+	nadOutputDir := r.artifactsDir
+
+	r.logNetworkAttachmentDefinition(virtCli, nadOutputDir)
+}
+
+func (r *KubernetesReporter) logNetworkAttachmentDefinition(virtCli kubecli.KubevirtClient, outputFolder string) {
+	networkAttachmentDefinitionsPath := filepath.Join(outputFolder, fmt.Sprintf("%d_networkAttachmentDefinitions.log", r.failureCount))
+	r.dumpK8sEntityToFile(virtCli, networkAttachmentDefinitionEntity, v1.NamespaceAll, networkAttachmentDefinitionsPath)
+}
```

Comment:
I tried to align with the structure of the functions that dumps k8s objects (e.g., sriovNetwork) and
if in the future there will be another object to log from the api group of `cni.cncf` it will be easier 
since you already have the abstraction of `logNetworkAttachmentDefinitionInfo` function.

Reply:
sounds good.